templates/infinispan-ephemeral.json

```diff
@@ -155,10 +239,25 @@
                         "fieldPath": "metadata.namespace"
                       }
                     }
+                  },
+                  {
+                    "name": "MGMT_USER",
+                    "value": "${MANAGEMENT_USER}"
+                  },
+                  {
+                    "name": "MGMT_PASS",
+                    "value": "${MANAGEMENT_PASSWORD}"
+                  },
+                  {
+                    "name": "APP_USER",
+                    "value": "${APPLICATION_USER}"
+                  },
+                  {
+                    "name": "APP_PASS",
+                    "value": "${APPLICATION_PASSWORD}"
                   }
                 ],
-                "image": "infinispan",
-                "imagePullPolicy": "Always",
+                "image": "${IMAGE}",
```

Comment:
Mainly for symmetry reasons...

We expect our users to use the persistent template in most of the cases. Since https://github.com/openshift/origin/pull/13210 has been fixed in `v3.6.0-alpha.2` we can not be sure that all of our clients will use OpenShift `3.6.0+`. So for *now*, we'd like to point to an absolute image in both templates (so that they look the same). Later on (in ~6 months?) we will be migrating to Image Stream approach in both those templates.

Does that sound reasonable?